packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -130,7 +129,23 @@ export const Dashboard = () => {
     isFetched,
   } = useDashboards(projectCode, entityCode, dashboardName);
   const [isExpanded, setIsExpanded] = useState(false);
-  const [exportModalOpen, setExportModalOpen] = useState(false);
+  const [exportModalOpen, setExportModalOpen] = useState<boolean>(false);
+  const [isSubscribed, setIsSubscribed] = useState<boolean>(false)
```

Comment:
@chris-pollard I think it makes a lot of sense for us to be tracking the `isSubscribed` on the `dashboard` itself. We can do this by adding `isSubscribed` to the response body for the `/dashboards` route in `tupaia-web-server`.

For now we can just have this always be `false`, but once we've implemented the mailing list it can be determined in the `tupaia-web-server`